discoverx/classification.py

```diff
@@ -115,8 +128,8 @@ def _get_or_create_classification_table_from_delta(self):
                 f"The classification table {self.classification_table_name} does not see to exist. Trying to create it ..."
             )
             (catalog, schema, table) = self.classification_table_name.split(".")
-            self.spark.sql(f"CREATE CATALOG IF NOT EXISTS {catalog}")
-            self.spark.sql(f"CREATE DATABASE IF NOT EXISTS {catalog + '.' + schema}")
+            self._create_databes_if_not_exists(catalog, schema)
```

Comment:
Should be easy to add a test for this as well. What do you think?

Reply:
I did add the tests for this, it’s just the codecov plugin that raises questionable warnings